.gitignore

```diff
@@ -14,3 +14,5 @@ cypress/screenshots
 .dev.vars
 /tests/http/http-client.private.env.json
 .wrangler
+test-dashboard.md
+t.ts
```

Comment:
What is this file?

Reply:
My mistake I'll remove that, I use `npx tsx t.ts` to run test scripts